Tests/UIKitTests/UIImageExtensionsTests.swift

```diff
@@ -132,6 +132,30 @@ final class UIImageExtensionsTests: XCTestCase {
         XCTAssertEqual(image?.size, size)
 
         XCTAssertEqual(image?.bytesSize, 787)
+
+        let scale: CGFloat = 5.0
```

Comment:
```suggestion
        let scale = CGFloat(5.0)
```

Reply:
@guykogus learning moment: what exactly does this provide in terms of improvement?  Is this just a coding preference?

Reply:
I think it's mostly a style choice, and it's what we try to consistently do.

It's also good practice because it's probably (although I doubt in this case) more efficient to use the standard initialisers. The reason that your code works, where you assign a `Double` to a `CGFloat`, is because `CGFloat` conforms to `ExpressibleByFloatLiteral`, so the compiler converts what you wrote to:
```Swift
let scale = CGFloat(floatLiteral: 5.0)
```
So better to save the compiler having to do that and just use the standard initialiser directly.